app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -421,12 +418,9 @@ private fun Rect.getIntersectPercentage(realSize: IntSize, other: Rect): Float {
 fun PocketStoriesCategories(
     categories: List<PocketRecommendedStoriesCategory>,
     selections: List<PocketRecommendedStoriesSelectedCategory>,
-    selectedTextColor: Color? = null,
-    unselectedTextColor: Color? = null,
-    selectedBackgroundColor: Color? = null,
-    unselectedBackgroundColor: Color? = null,
-    onCategoryClick: (PocketRecommendedStoriesCategory) -> Unit,
     modifier: Modifier = Modifier,
```

Comment:
I am curious does Android Studio complain about the `modifier` position here? I believe I have seen it recommend that `modifier` should be the second optional parameter if the composable function accepts a modifier parameter. If it does complain, let's fix it here as well.

Reply:
It complains if it isn't the first optional parameter

Reply:
Oh okay, then everything is fine then.